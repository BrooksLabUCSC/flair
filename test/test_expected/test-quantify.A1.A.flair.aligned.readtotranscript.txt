HISEQ:1287:HKCG7BCX3:1:1104:17934:72463	HISEQ:1287:HKCG7BCX3:1:1112:9533:55818_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1112:9533:55818	HISEQ:1287:HKCG7BCX3:1:1112:9533:55818_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1114:17507:23093	HISEQ:1287:HKCG7BCX3:1:1101:9136:21962_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1109:6713:22942	HISEQ:1287:HKCG7BCX3:1:1101:9136:21962_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1102:9094:10890	HISEQ:1287:HKCG7BCX3:1:1101:9136:21962_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1109:19527:97868	HISEQ:1287:HKCG7BCX3:1:1110:2082:85316_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1104:6374:92450	HISEQ:1287:HKCG7BCX3:1:1113:9124:44339_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1107:14080:13641	HISEQ:1287:HKCG7BCX3:1:1113:9124:44339_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1106:5425:16663	HISEQ:1287:HKCG7BCX3:1:1104:2444:51920_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1104:5295:15725	HISEQ:1287:HKCG7BCX3:1:1104:2444:51920_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1113:17458:44365	HISEQ:1287:HKCG7BCX3:1:1104:2444:51920_chr12:25205000
HISEQ:1287:HKCG7BCX3:1:1111:10954:88390	HISEQ:1287:HKCG7BCX3:1:1107:7610:49233_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1101:10287:53003	HISEQ:1287:HKCG7BCX3:1:1107:7610:49233_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1102:6372:22762	HISEQ:1287:HKCG7BCX3:1:1107:7610:49233_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1105:4350:92846	HISEQ:1287:HKCG7BCX3:1:1107:7610:49233_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1104:14867:57668	HISEQ:1287:HKCG7BCX3:1:1101:1676:64370_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1103:12415:65429	HISEQ:1287:HKCG7BCX3:1:1101:1676:64370_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1101:2996:10381	HISEQ:1287:HKCG7BCX3:1:1101:1676:64370_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1111:8082:26682	HISEQ:1287:HKCG7BCX3:1:1113:19423:71141_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1113:20069:3952	HISEQ:1287:HKCG7BCX3:1:1113:19423:71141_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1113:19423:71141	HISEQ:1287:HKCG7BCX3:1:1113:19423:71141_chr12:25206000
HISEQ:1287:HKCG7BCX3:1:1106:17802:96954	HISEQ:1287:HKCG7BCX3:1:1114:18197:37779_chr12:25207000
HISEQ:1287:HKCG7BCX3:1:1106:14742:75120	HISEQ:1287:HKCG7BCX3:1:1114:18197:37779_chr12:25207000
HISEQ:1287:HKCG7BCX3:1:1106:5073:68601	HISEQ:1287:HKCG7BCX3:1:1114:18197:37779_chr12:25207000
HISEQ:1287:HKCG7BCX3:1:1104:16146:58913	HISEQ:1287:HKCG7BCX3:1:1102:1600:74434_chr12:25207000
HISEQ:1287:HKCG7BCX3:1:1108:14338:16394	HISEQ:1287:HKCG7BCX3:1:1102:1600:74434_chr12:25207000
HISEQ:1287:HKCG7BCX3:1:1103:10178:25804	HISEQ:1287:HKCG7BCX3:1:1105:13763:58932_chr12:25233000
HISEQ:1287:HKCG7BCX3:1:1106:11735:87772	ENST00000311936.8_ENSG00000133703.12
HISEQ:1287:HKCG7BCX3:1:1102:16745:64007	ENST00000311936.8_ENSG00000133703.12
HISEQ:1287:HKCG7BCX3:1:1101:6957:64687	ENST00000557334.5_ENSG00000133703.12
HISEQ:1287:HKCG7BCX3:1:1105:8148:44134	HISEQ:1287:HKCG7BCX3:1:1107:13669:8305_ENSG00000108654.15
HISEQ:1287:HKCG7BCX3:1:1113:10231:17490	ENST00000451605.1_ENSG00000125991.19
HISEQ:1287:HKCG7BCX3:1:1109:16615:94483	ENST00000451605.1_ENSG00000125991.19
HISEQ:1287:HKCG7BCX3:1:1104:18450:12186	ENST00000451605.1_ENSG00000125991.19
HISEQ:1287:HKCG7BCX3:1:1105:14367:86687	ENST00000451605.1_ENSG00000125991.19
HISEQ:1287:HKCG7BCX3:1:1111:16109:44469	ENST00000451605.1_ENSG00000125991.19
