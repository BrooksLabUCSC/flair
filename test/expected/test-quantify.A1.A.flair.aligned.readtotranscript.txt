m54284U_201123_202240/35980470/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201123_202240/161614162/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201029_221949/168954339/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201028_155127/23397694/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201029_221949/132646592/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201123_202240/93259725/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201028_155127/149096461/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201123_202240/88736174/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201123_202240/150537051/ccs	ENST00000311936.8_ENSG00000133703.12
m54284U_201123_202240/128977546/ccs	m54284U_201123_202240/28902124/ccs-0_ENSG00000133703.12
m54284U_201029_221949/46202895/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/179242613/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/12584637/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/8062359/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/8587947/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/27133410/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/28640675/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/58917622/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/139264285/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/160564925/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/23857676/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/40700251/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/63571314/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/132581025/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/164104409/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/10551616/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/21692686/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/58590413/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/162924902/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/8587143/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/30278314/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/37029660/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/164955559/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/60031506/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/161418069/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/101254018/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/152962448/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/42142340/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/83101238/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/150865324/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/40045358/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/96863499/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/164365532/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/15927302/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/16190132/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/20711759/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/50858125/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/87361421/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/94438484/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/102891641/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/132841603/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/144310340/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/160434063/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/176620628/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/86376954/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/70257226/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/10291775/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/89063752/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/13763028/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/33425269/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/37487575/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/32506259/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/107282651/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201029_221949/89524058/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/80808532/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/71894380/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/104006316/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/95486286/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/119276889/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201028_155127/60228627/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/5374797/ccs	ENST00000225792.10_ENSG00000108654.15
m54284U_201123_202240/146473458/ccs	ENST00000581230.5_ENSG00000108654.15
m54284U_201028_155127/160563992/ccs	ENST00000581230.5_ENSG00000108654.15
m54284U_201028_155127/120587977/ccs	m54284U_201123_202240/160760815/ccs-0_ENSG00000108654.15
m54284U_201123_202240/168560882/ccs	m54284U_201123_202240/132317757/ccs-0_ENSG00000108654.15
m54284U_201123_202240/171509611/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201028_155127/82707233/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201123_202240/146474652/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201029_221949/141821598/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201028_155127/44894949/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201029_221949/176490917/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201029_221949/60688126/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201123_202240/37292085/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201029_221949/54396423/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201029_221949/162399753/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201123_202240/7996411/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201028_155127/179241646/ccs	ENST00000348547.6_ENSG00000125991.19
m54284U_201028_155127/177406433/ccs	ENST00000348547.6_ENSG00000125991.19
